docs/user-guide/tutorials/offline_vs_online.md

```diff
@@ -18,9 +20,9 @@ First, let's use the `predict`-command to download the data and compute embeddin
 Open a terminal in the folder where you installed *eva* and run:
 
 ```
-PRETRAINED=false \
-EMBEDDINGS_ROOT=./data/embeddings/dino_vits16_random \
-eva predict --config configs/vision/dino_vit/offline/bach.yaml
+export MODEL_NAME=universal/vit_small_patch16_224_random
+export EMBEDDINGS_ROOT=./data/embeddings/dino_vits16_random
```

Comment:
end line 23, 24 with `\` to set env variables only for the command

Reply:
I did that on purpose to avoid redundancy :) elsewise during the *offline* tutorial you have to repeat exporting those  multiple times.

Reply:
i would still avoid setting them globally. This way you are guranteed to have the default setup if no env variables are specified

Reply:
done